### PR TITLE
Finetune Panelite dependencies

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Adds more dependencies to the list

### CODE REVIEW

File: scripts/panelite_dependencies.json [hunk 1]
```diff
@@ -1,5 +1,68 @@
 {
     "developer_guide/Developing_Custom_Models.ipynb": [],
+    "gallery/apis/stocks_altair.ipynb": ["panel"],
```
Comment (philippjfr) [inline scripts/panelite_dependencies.json:3]: These are about to basically all go away (or rather move under how_to) so I'll rework this.

**Comment by MarcSkovMadsen** [comment]
I've now been through all the notebooks in Panelite. I've identified and opened issues where ever experienced. I've listed the individual dependencies of the notebooks in Panelite. This should help improve the user experience in Panelite a lot.

In later PRs I hope to 

- Update the README and docs to make Panelite a bit more discoverable.
- Add some comment at the top of the Panelite notebooks that Panelite is experimental, that this notebook has the following know issues ... and that issues can be reported at ...
- Setup some (semi-)automated build and test of Panelite that can help to identify and fix issues.

**Comment by MarcSkovMadsen** [comment]
The ubuntu 3.8 test fails. Its not clear to me why

image

**Comment by philippjfr** [comment]
Not a real failure, the machines sometimes run out of memory.

**Comment by philippjfr** [comment]
This is now superseded by other PRs right?

**Comment by MarcSkovMadsen** [comment]
Yes

**Pull request closed without merging**